### ISSUE TITLE
Package for PyPI

### ISSUE THREAD

**Issue by eleurent** [eleurent/highway-env#143]
* [x] Release a new version with latest changes
* [x] Follow guide from: https://packaging.python.org/tutorials/packaging-projects/
* [x] Update requirements of projects that depend on highway-env

**Comment by stefanbschneider** [comment]
Hi, saw that the GIFs in your readme are shown nicely also on the [`highway-env` PyPi page](https://pypi.org/project/highway-env/). Did you have to do any trick to get that working?

I also have a [project on GitHub with GIFs in the readme](https://github.com/CN-UPB/DeepCoMP) and publish new releases automatically on PyPi with GitHub actions. Unfortunately, my [GIFs always break on PyPi and are not shown correctly](https://pypi.org/project/deepcomp/).

I can't seem to find any difference in how embed the GIFs in my readme vs. how you did it. Any idea what the problem could be? Thanks! I know this is a bit off topic...

Btw: Very nice project!

**Comment by eleurent** [comment]
Hi! Well that is very strange indeed.
The pypi gif links were initially broken in my first attempt (same result as yours), because I was using relative links in the readme, so I simply moved to absolute links in d5dc1a5364d42b8912558e3eb7ef99e753cafc7e and it worked right away.
But I cannot find any difference either in the gif url between your readme vs mine...

However, I noticed something: when I click to the gif of highway-env, it is directly served in my browser (I'm using Mozilla Firefox), whereas the gif of DeepCoMP triggers the file download prompt instead. 
This is probably the reason why pypi handles the two cases differently as well, but I wonder what the root cause is... File size?
